src/main/java/soconnect/logic/parser/ParserUtil.java

```diff
@@ -26,6 +27,7 @@ public class ParserUtil {
     /**
      * Parses {@code oneBasedIndex} into an {@code Index} and returns it. Leading and trailing whitespaces will be
      * trimmed.
+     *
      * @throws ParseException if the specified index is invalid (not non-zero unsigned integer).
```

Comment:
```suggestion
     * @throws ParseException If the specified index is invalid (not non-zero unsigned integer).
```

Reply:
Thanks for reviewing, I have made the changes!